README.md

```diff
@@ -18,27 +18,32 @@ Based on [redux-saga-rxjs](https://github.com/salsita/redux-saga-rxjs) by Salsit
 npm install store-saga --save
 ```
 
-Write a saga:
+Write a saga (you _don't_ have to import and specify all those types, but they help to spot errors earlier):
+
 ```ts
+import {Observable} from 'rxjs/Observable';
+import {Action} from '@ngrx/store';
 import {createSaga} from 'store-saga';
 
-export const increment = createSaga(function(){
-  return iteration$ => iteration$
-    .filter(iter => iter.action.type === 'DECREMENT')
-    .map(() => {
-      return { type: 'INCREMENT'}
-    });
+export const incrementProvider = createSaga(function() {
+  return function incrementSaga(iteration$): Observable<Action> {
+    return iteration$
+      .filter(iter => iter.action.type === 'DECREMENT')
+      .map(() => {
```

Comment:
Just edited my reply, I guess too late :smile: ... we have that import already. Everytime we save the file, red squiggles disappear for a moment, then they're back again. We only see those on all RxJs operators and observable static methods, although we import them:

```ts
import 'rxjs/add/observable/of';
import 'rxjs/add/operator/catch';
```

We still have RxJs type definitions inside `typings` folder, although in more recent version the library itself in `node_modules` comes with its own type definitions ... don't know if that is problem.... we'll try uninstalling those typings.